data/json/monsters.json

```diff
@@ -5050,7 +5050,7 @@
     "id" : "mon_dark_wyrm",
     "name": "dark wyrm",
     "species":"MUTANT",
-    "default_faction":"mutant",
+    "default_faction": "jabberwock",
```

Comment:
Jabberwock faction is hostile to own members and so shouldn't be used for critters that attack in groups.